recipes/anyqt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "AnyQt" %}
+{% set version = "0.0.6" %}
+{% set sha256 = "40a77fd166cb2d9ba04c01b892fa646d93affedfea96ad8411995b0b7cd5db9d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+  script:
+    - pip install --no-deps --no-index --no-cache ./
```

Comment:
I removed the use of pip. Now using the standard `setup.py install --single-version...`